Add OpenRGB Guide (#56)
Vanilla-OS/handbook: _posts/2023-01-24-install-openrgb.md

---
title: Install OpenRGB
description: Learn how to install OpenRGB on Vanilla OS.
date: 2023-01-20
layout: article
authors: 
    - MonsterObserver
published: true
---

## Introduction

[**OpenRGB**](https://openrgb.org/) is an essential application for managing your RGB appliances. This guide will help you install the Flatpak and AppImage of OpenRGB correctly.

## Flatpak

### Prerequisites

- Flatpak: You can find out how to install it [**here**](/2022/12/09/install-flatpaks.html).

### Installing OpenRGB

- You can install OpenRGB from the "Gnome Software" application. Alternatively, you can install it from the command line with the following command:-

```bash
flatpak install flathub org.openrgb.OpenRGB
```

## AppImage

### Prerequisites

> - AppImage Support: If you have enabled AppImage in the "First Setup" when installing Vanilla OS, this step isn't necessary. Or else, run this command:-

Author: ```suggestion
- AppImage Support: If you have enabled AppImage in the "First Setup" when installing Vanilla OS, this step isn't necessary. If you didn't enable it, run the following command:-
```

> - AppImage Support: If you enabled AppImage in the "First Setup" when installing Vanilla OS, you won't have to do anything. If you didn't enable AppImage, run this command:-

Reviewer: ```suggestion
- AppImage Support: If you have enabled AppImage in the "First Setup" when installing Vanilla OS, this step isn't necessary. Else, run this command:-
```

```bash
sudo abroot exec apt install libfuse2
```

> - After the installation is complete reboot the system. You can do this by typing `reboot`.

Reviewer: ```suggestion
- After the successful installation, reboot your system. You can do this by typing `reboot`.
```

### Installing OpenRGB

- Install "Linux 64-bit" from the "Linux Binaries (AppImage)" section [**here**](https://gitlab.com/CalcProgrammer1/OpenRGB/-/releases/permalink/latest#Linux-64-bit).

> - Go to the directory where the AppImage is installed and make it an executable with these commands:-

Author: ```suggestion
- Go to the directory where the AppImage is installed and make it an executable with the following commands:-
```

> - Go to the directory where the AppImage is installed and make it an executable with these commmands:-

Reviewer: ```suggestion
- Go to the directory where the AppImage is installed and make it an executable with these commands:-
```
